toc/elections/2022/Readme.md

```diff
@@ -0,0 +1,85 @@
+# 2022 CFF TOC Election Guide
+
+
+## Overview
+
+
+Each year, the CFF technical community holds an election for open seats on the 
+Technical Oversight Committee (TOC). The rules governing this election are set in the 
+[CFF's project charter](../../../governing-board/charter.md) (See section 7(b) and 7(e)
+for the relevant details).
+
+
+This guide exists to serve as a guide to this year's election process.
+
+
+## Schedule
+
+
+| Date                       | Event                    |
+| -------------------------- | ------------------------ |
+| May 11                   | Announcement of Election (at least 4 weeks before results) |
+| May 11 through May 28  | Candidate nomination period (at least 2 weeks long and ending 2 weeks before results) |
+| May 31     | Election Begins via email ballots (alow 2 work days to prepare election system) |
+| June 15     | Election Closes (at least 2 weeks after election begins) |
+| *June 17*   | Announcement of Results (at least 2 work days after election ends) |
+
+
+## Candidate Processes
+
+
+**Nominations**
+
+
+Every eligible voter can nominate candidates for the TOC, and we encourage you to do so. If you are 
+eligible to serve on the TOC, you can self nominate! If you want to nominate someone else, do so as 
+well! 
+
+
+You can nominate someone for the TOC by [submitting an issue using this template](https://github.com/cloudfoundry/community/issues/new?assignees=&labels=election&template=toc-candidate-nomination.md&title=TOC+Candidate+Nomination+for+%5BPerson+Name%5D). 
+
+
+The deadline is May 28th to be nominated (and for the nominee to indicate acceptance).
+
+
+It is strongly recommended that you confirm that a nominee is interested and willing
+to accept the nomination prior to submitting the nomination issue.
+
+
+**Confirming Nominee Eligibility**
+
+
+Once a nomination is received, the CFF staff will contact the nominee to confirm acceptance
+of the nomination. If the nominee accepts, they will be added to the list of nominees at the bottom
+of this file. The nomination issue will be closed, noting if the nominee accepted or declined the 
+nomination.
+
+
+## Voting Process
+
+
+The election will be conducted using a time-limited [Condorcet](https://civs.cs.cornell.edu/rp.html) ranking 
+on [CIVS](http://civs.cs.cornell.edu/) using the Schulze method. 
+
+
+Voters will receive an email with a ballot link. Voters will have until the end of the election cycle 
+to submit their ballot.
```

Comment:
I'd like us to explore using https://github.com/elekto-io/elekto as the election system for this cycle, which Kubernetes has been using for its [Steering Committee elections](https://groups.google.com/g/kubernetes-dev/c/Ds-A499pzg4) and which Knative also [started using last year](https://knative.dev/blog/steering/toc-2021-election-announcement/). We ran into significant confusion with the email exchanges and the once-a-day syncing cycle from the CIVS service last time.

Reply:
BTW, I started getting Elekto to build and run on CF on my fork at https://github.com/emalm/elekto/tree/run-with-cf. I've gotten it to stage and to connect to a bound MySQL DB automatically, but I haven't tried configuring the GitHub settings yet.

Reply:
Hah, I kept playing around with this, and it's now running at https://elekto-emalm.apps.dhaka.cf-app.com, with the election config repo at https://github.com/emalm/elekto.meta.test. @christopherclark, I made you both an admin and a voter for the "name the app" election that it's running right now, if you wanted to take a look at it.